tests/functional/src/main/java/org/mule/functional/classloading/isolation/maven/MuleMavenMultiModuleArtifactMapping.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.functional.classloading.isolation.maven;
+
+import static java.lang.Thread.currentThread;
+import static org.apache.commons.lang.StringUtils.isEmpty;
+import static org.mule.runtime.core.util.PropertiesUtils.loadProperties;
+
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * Mule default implementation for getting modules based on artifactIds.
+ *
+ * @since 4.0
+ */
+public class MuleMavenMultiModuleArtifactMapping implements MavenMultiModuleArtifactMapping
+{
+    public static final String MAVEN_MODULE_MAPPING_PROPERTIES = "maven-module-mapping.properties";
```

Comment:
As far as I know there is no way to get this when the test is executed, the only plugin that holds this information but it is not exposed is the reactor plugin. I found some tools (with grep and other plugins) to get this list (sort of, because I just need the relative path) but I still don't know how to improve this, maybe there has to be another plugin (like reactor) that builds this list and sets it as part of the maven build context so them it can be accessed as system property or something else.
But no, there is no plan yet to get rid of this. At least for the next weeks.